### DISCUSSION TITLE
Zasób do masowej edycji cen w licytacjach

### DISCUSSION THREAD

**Discussion by pawelosajda** [allegro/allegro-api#4639] [Suggestion - verification]
Dzień dobry w tym wątku zgłaszałem potrzebę zosobu do masowej edycji cen w licytacjach https://github.com/allegro/allegro-api/issues/2153 Czy od tego czasu coś się zmieniło w planach dotyczących dodania takiego zasobu? Czy może trwają pracę nad możliwością masowej zmiany ceny w licytacjach?

**Comment by MartaNowaczyk** [comment]
Na tę chwilę nic się nie zmieniło. Wątek przenoszę do naszej zakładki [Discussions](https://github.com/allegro/allegro-api/discussions), gdzie zbieramy sugestie. Dzięki temu również inni użytkownicy będą mogli na nią zagłosować, co dodatkowo określi, jak bardzo priorytetowa jest to potrzeba. Więcej o zakładce Discussions znajsziesz w [naszym newsie](https://github.com/allegro/allegro-api/issues/4337).
